docs/riff_streaming_inmemory-gateway.md

```diff
@@ -8,7 +8,10 @@ title: "riff streaming inmemory-gateway"
 
 ### Synopsis
 
-<todo>
+The in-memory Gateway encapsulates the address of a streaming gateway and a no-op Provisioner instance.
+
+The no-op Provisioner is responsible for resolving the coordinates of the in-memory topic in streaming gateway.
+The streaming gateway coordinates and standardizes reads and writes to a number of message brokers, which includes an in-memory broker implementation.
```

Comment:
This sentence (and this is true for all gateways) is misleading IMO. A __single__ gateway doesn't deal with a "number of message brokers" (plural).
Also, as soon as we're dealing with a special kind of gateway in documentation speech (such as in memory here), we shouldn't "generalize back", so "which includes ..." just adds confusion IMHO.